src/bin/pg_autoctl/cli_create_drop_node.c

```diff
@@ -266,7 +269,9 @@ cli_create_monitor_getopts(int argc, char **argv)
 		{ "nodename", required_argument, NULL, 'n' },
 		{ "listen", required_argument, NULL, 'l' },
 		{ "auth", required_argument, NULL, 'A' },
-		{ "help", no_argument, NULL, 0 },
```

Comment:
help was not explicitly handled before. now we are removing support for --help option. Is this intentional ?